docs-developer/deploying.md

```diff
@@ -25,7 +25,7 @@ To be able to get the changed locales quickly, this command can be used
 (assuming that `upstream` is the remote you use for this repository):
 
 ```
-git fetch upstream && git diff --name-only upstream/main..upstream/l10n | awk -F '/' '{printf $2 ", "}'; echo
+git fetch upstream && git diff --name-only upstream/main...upstream/l10n | awk -F '/' '{printf $2 ", "}'; echo
```

Comment:
Excerpt from the man page for `git diff`:
```
       git diff [<options>] <commit>...<commit> [--] [<path>...]
           This form is to view the changes on the branch containing and up to the second <commit>, starting at a common ancestor of both
           <commit>.  git diff A...B is equivalent to git diff $(git merge-base A B) B. You can omit any one of <commit>, which has the same
           effect as using HEAD instead.
```

This is basically the opposite of `man gitrevisions` (which is used in `git log` for example), which makes things especially confusing :D

Reply:
I see. Yeah, it's a bit confusing indeed. Thanks for fixing it!